src/server/boot/03-offline-login.js

```diff
@@ -37,12 +38,23 @@ export default function (app) {
     }),
   )
 
-  app.use('/login/password', passport.authenticate('basic'), (req, res) => {
-    const responseType = req.accepts(['json', 'html']) || 'json'
-    if (responseType === 'json') {
-      res.status(200).json({ message: 'Login successful' })
-    } else {
-      res.redirect(303, '/')
-    }
-  })
+  app.use(
+    '/login/password',
+    passport.authenticate('basic'),
+    async (req, res) => {
```

Comment:
since this is an async handler function, you should use app.wrap for it, to make sure any exceptions get handled properly by express (it doesn't await promises by itself)

Reply:
Done